test/modules/toolbox/components/shareButton/ShareDialogContent.test.js

```diff
@@ -67,6 +67,16 @@ describe('ShareDialogContent', () => {
 		expect(copyButton).toBeFalsy();
```

Comment:
Falsy/truthy tests are weak tests. Test expected presences or absence of element by using
```
expect(element.shadowRoot.querySelectorAll('.share_item .share_api')).toHaveSize(1);
expect(element.shadowRoot.querySelectorAll('.share_item .share_copy')).toHaveSize(0);
```